net.crossfireatlas.crossfire.yaml

```diff
@@ -0,0 +1,26 @@
+app-id: net.crossfireatlas.crossfire
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: 
+command: net.crossfireatlas.crossfire.start.sh
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: 
+      - /usr/lib/sdk/openjdk11/install.sh
```

Comment:
it's actually surprising the build didn't fail beucase this is missing. See comment about about the empty sdk-extension.

Reply:
I added sdk-extensions: org.freedesktop.Sdk.Extension.openjdk11
to the yaml file. So, this resolves this comment?